tests/features/joinup_discussion/discussion.invite.feature

```diff
@@ -0,0 +1,144 @@
+@api
+Feature: Invite members to subscribe to discussions
+  In order to promote discussion of topics
+  As a discussion author or moderator
+  I need to be able to invite users to discussions
+
+  @email
+  Scenario: Invite members to a discussion
+    Given users:
+      | Username            | E-mail                   | First name | Family name  |
+      | Viktor Bhattacharya | whackybhatta@example.com | Viktor     | Bhattacharya |
+      | Vikentiy Rozovsky   | v.rozovsky@example.com   | Vikentiy   | Rozovsky     |
+      | Roxanne Stavros     | whackyroxy@example.com   | Roxanne    | Stavros      |
+      | Rodrigo Villanueva  | r.villanueva@example.com | Rodrigo    | Villanueva   |
+    And the following collection:
+      | title             | The Siphon Community |
+      | description       | We just love siphons |
+      | state             | validated            |
+    And the following collection user membership:
```

Comment:
Discussed with the functional team. They say solutions should be included too. I see that are included in code but we have to do it also in tests.